README.md

```diff
@@ -1,4 +1,53 @@
-# devtools-protocol
+# DevTools protocol
 
-This is the repository for devtools-protocol. You're welcome to
-[[contribute]](CONTRIBUTING.md)!
+This repository is for resources and discussion around aligning the developer tools wire protocol used by various browsers, JavaScript egines and debugging tools.
+
+## Goals
```

Comment:
Thoughts on adding a goal around collaborating on test coverage of overlapping APIs? Maybe add that to the key use cases section? I'm imagining something like contributions to web-platform-tests in a new "DevTools-protocol" area

Reply:
Absolutely.  I already filed #7 to discuss this, but I can mention it here as a goal too.